app/assets/stylesheets/semantic-ui/globals/_variables.scss

```diff
@@ -1,4 +1,37 @@
-$import-google-fonts: true !default;
-$font-url: 'https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&subset=latin,latin-ext' !default;
-$font-name: 'Lato' !default;
-$font-family: $font-name, 'Helvetica Neue', Arial, Helvetica, sans-serif !default;
+$font-url: 'proxima-nova';
+$import-google-fonts: 'proxima-nova';
```

Comment:
How is this variable used? Changing from `true` to `proxima-nova` is kind of strange. `false` would make more sense if it is used as the name suggest